docs/cpp/cmake-linux.md

```diff
@@ -78,57 +78,47 @@ sudo apt-get install build-essential gdb
 
-This creates a hello world CMake project containing `main.cpp`, `CMakeLists.txt` (which tells the CMake tools how to build your project), and a folder named `build` for your build files:
+![CMake Select presets](images/cpp/select-cmake-preset.png)
 
-![Project contents](images/cpp/cmake-project-contents.png)
+#### Configure Using CMake Kits
+If your project does not have a `CMakePresets.json` file, you need to use kits. A kit represents a toolchain, which is the compiler, linker, and other tools used to build your project.
 
-### Select a kit
+To scan for kits:
 
-Before you can use the CMake Tools extension to build a project, you need to configure it to know about the compilers on your system. Do that by scanning for 'kits'. A kit represents a toolchain, which is the compiler, linker, and other tools used to build your project. To scan for kits:
-
-1. Open the Command Palette (`kb(workbench.action.showCommands)`) and run **CMake: Select a Kit**. The extension will automatically scan for kits on your computer and create a list of compilers found on your system.
+1. Open the Command Palette (`kb(workbench.action.showCommands)`) and run **CMake: Select a Kit**. The extension automatically scans for kits on your computer and creates a list of compilers found on your system.
 
 1. Select the compiler you want to use. For example, depending on the compilers you have installed, you might see something like:
 
     ![Select the kit](images/cpp/cmake-selectkit.png)
 
-## Configure Hello World
-
-There are two things you must do to configure your CMake project: select a kit (which you just did) and select a variant.
-
-The kit you selected previously is shown in the **Project Status** view in the CMake Tools sidebar. For example:
+The kit you selected previously is shown in the **Project Status** section in the CMake Tools view. For example:
 
 ![Selected kit in Sidebar](images/cpp/cmake-kit.png)
 
-To change the kit, you can click on the kit in the **Project Status** view in the CMake Tools sidebar, or run the **CMake: Select a Kit** command again from the Command Palette. If you don't see the compiler you're looking for, you can edit the `cmake-tools-kits.json` file in your project. To edit the file, open the Command Palette (`kb(workbench.action.showCommands)`) and run the **CMake: Edit User-Local CMake Kits** command.
+To change the kit, you can select the kit in the **Project Status** section in the CMake Tools view, or run the **CMake: Select a Kit** command again from the Command Palette. If you don't see the compiler you're looking for, you can edit the `cmake-tools-kits.json` file in your project. To edit the file, open the Command Palette (`kb(workbench.action.showCommands)`) and run the **CMake: Edit User-Local CMake Kits** command.
 
-### Select a variant
+##### Select a variant
```

Comment:
Is this section a subsection of "Configure using CMake Kits"? It's weird to have a single subsection.

Reply:
I added it to be one large section